script/release/kie-upgrade-branch-auxiliary-reps.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash -e
```

Comment:
Please consider using
`set -euxo pipefail`
Which makes the bash scripts more reliable overall. See [this article](https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/) for details.